models/Comment.js

```diff
@@ -23,6 +23,18 @@ class Comment {
 
     return result.rows[0]
   }
+
+  static async deleteById(id) {
+    const result = await db.query(
+      `
+      DELETE FROM comments
+      WHERE id = $1
```

Comment:
I'm guessing you changed the column name in the seed to be id rather than comment_id?